config.json

```diff
@@ -27,6 +27,16 @@
         "logic"
       ]
     },
+    {
+      "uuid": "",
```

Comment:
You need to add a unique UUID. Use `configlet uuid` to generate one.

Reply:
Also, make sure when reading the Travis build output that you look for the earliest error; sometimes these cause the later ones and are the real problem. From this particular failure:
```Bash
$ ./bin/configlet lint .

-> The exercise 'reverse-string' was found in config.json, but does not have a UUID.

The command "./bin/configlet lint ." exited with 1.
```